src/search/vectorStoreManager.ts

```diff
@@ -121,15 +132,16 @@ export default class VectorStoreManager {
     this.dbOps.onunload();
   }
 
-  public async getOrInitializeDb(embeddingsAPI: Embeddings): Promise<Orama<any>> {
```

Comment:
We shouldn't have another place to initialize DB. `initialize` method be the one that instantiate db